[CORL-829] Bump line-height on select fields

nick-funk opened the pull request:
Bump line-height on select to account for hanging font characters

## What does this PR do?

Prevents characters like "g" or "j" being cut off at the bottom of the select text area.

## How do I test this PR?

Head to Admin > Configure > General.
See that the select drop down is no longer cutting off the text.

<img width="369" alt="image" src="https://user-images.githubusercontent.com/5751504/71826172-900a3780-305a-11ea-9622-50fb2d6535ba.png">

nick-funk committed:
Bump line-height on select to account for hanging font characters

Prevents characters like "g" or "j" being cut off at the
bottom of the select.

CORL-829

tessalt reviewed (commented):
I was thinking we might want to add another variable for this line height, since it's come up for me in a few more places

nick-funk commented:
> I was thinking we might want to add another variable for this line height, since it's come up for me in a few more places

Makes sense to me, I'll try and pick a reasonably generic name for the line height.

nick-funk committed:
Create new tall line height

Use it for the select boxes to allow for dangling
characters like 'g' and 'j'.

CORL-829
